doc/contributing.rst

```diff
@@ -493,11 +420,11 @@ General commit rules
 --------------------
 
 1. Important, critical bugfixes that have a tiny chance of breaking
-   somebody's code should be backported to the latest stable release
-   branch (currently 1.4.x) and maybe also all the way back to the 1.0.x branch.
-   The commit message should contain the tag ``[backport]`` for "backport to the latest
-   stable release" and the tag ``[backport:$VERSION]`` for backporting back to the
-   given $VERSION (and all newer releases).
+   somebody's code should be backported to the latest stable release branch
```

Comment:
Maybe we should remove this back porting thing and revisit when we care?

Reply:
I personally don't care either way, so I will remove it for now